runestone/selectquestion/js/selectone.js

```diff
@@ -121,14 +125,24 @@ export default class SelectOne extends RunestoneBase {
             self.containerDiv = res.question.containerDiv;
             self.realComponent.selectorId = selectorId;
         } else {
-            ///////////////////////////
             if (data.toggle) {
+                var toggleOptions = data.toggleOptions;
+                var toggleLabels;
+                if (toggleOptions.includes("labels{")) {
+                    toggleLabels = toggleOptions.slice(toggleOptions.indexOf("labels{") + 7, toggleOptions.indexOf("}")).split(",");
+                    toggleOptions = toggleOptions.slice(0, toggleOptions.indexOf(",labels{"));
```

Comment:
I think this is your problem.  You are slicing from the beginning of the string to labels.

If I have time later I can try to pull and debug in the browser but this seems like the most likely culprit.

Reply:
I think the cut off occurs before it gets to this point. Even without including labels, the presence of a space seems to cause the text following that space being dropped:
![image](https://user-images.githubusercontent.com/40499850/118011089-a050b780-b31d-11eb-81cf-72bc95391be0.png)
![image](https://user-images.githubusercontent.com/40499850/118011182-b9f1ff00-b31d-11eb-8b10-e7e06c693e15.png)
![image](https://user-images.githubusercontent.com/40499850/118011282-d8f09100-b31d-11eb-99ff-8bf17ab023bc.png)